docs/hazmat/primitives/asymmetric/ec.rst

```diff
@@ -47,6 +47,16 @@ Elliptic Curve Signature Algorithms
     :param algorithm: An instance of
         :class:`~cryptography.hazmat.primitives.hashes.HashAlgorithm`.
 
+    :param bool deterministic_signing: A boolean flag defaulting to ``False``
+        that specifies whether the signing procedure should be deterministic
+        or not, as defined in :rfc:`6979`.
```

Comment:
Docs should say that this has no impact on verification.

Reply:
Fixed. The new docstring now says:
```rst
    :param bool deterministic_signing: A boolean flag defaulting to ``False``
        that specifies whether the signing procedure should be deterministic
        or not, as defined in :rfc:`6979`. This only impacts the signing
        process, verification is not affected (the verification process
        is the same for both deterministic and non-deterministic signed
        messages).
```